org.opent2t.sample.binaryswitch.superpopular/org.opent2t.sample.binaryswitch.superpopular.xml

```diff
@@ -2,6 +2,8 @@
 <!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN" "http://standards.freedesktop.org/dbus/introspect-1.0.dtd"[]>
 <node>
   <interface name="org.opent2t.sample.binaryswitch.superpopular">
+    <description>OpenT2T SuperPopular Binary Switch</description>
+
```

Comment:
We should try to be... consistent, yes! Please follow up with a PR with any fixes you want to propose... we should constantly be tidying. For now, I will #WontFix but look forward to a PR from you.